ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -0,0 +1,29 @@
+namespace ConwaysGameOfLife.Tests
+{
+    public static class UniverseHelper
+    {
+        public static bool UniverseContentsAreEqual(Universe obj1, Universe obj2)
+        {
+            var isSizeSame = (obj1.GridWidth == obj2.GridWidth) && (obj1.GridLength == obj2.GridLength);
+            if (!isSizeSame)
+            {
+                return false;
+            }
+            // test location using new method - for loop using index
+            for (var x = 0; x < obj1.GridLength; x++)
+            {
+                for (var y = 0; y < obj1.GridLength; y++)
+                {
+                    if (obj1.GetCellAtLocation(x, y) != obj2.GetCellAtLocation(x, y))

```

Comment:
this is a good start, adding a note here as a reminder to refactor this tomorrow. Options:
1. Add another helper method to check if 2 cells are equivalent
2. Do a manual check if cell state 

Also thought of a better name for this function - UniversesAreEquivalent